From 6ae3f369548c9a4d86621865f664f65033a52789 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 5 Dec 2023 18:31:25 +0000
Subject: [PATCH 0764/1085] arm: dt: bcm2712: Correct the size of the register
 range for MOP

The Mop covers 0x28 bytes of registers, so ensure the range is
defined appropriately.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712.dtsi
@@ -135,7 +135,7 @@
 
 		mop: mop@7c500000 {
 			compatible = "brcm,bcm2712-mop";
-			reg = <0x7c500000 0x20>;
+			reg = <0x7c500000 0x28>;
 			interrupt-parent = <&disp_intr>;
 			interrupts = <1>;
 			status = "disabled";
